src/main/java/com/myapp/ordernow/models/Order.java

```diff
@@ -0,0 +1,49 @@
+package com.myapp.ordernow.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
+
+@Entity
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Table(name = "Orders")
+public class Order {
+    private String name;
+    private String address;
+    private String phone;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
use @data  annotation to let springboot take care of the getters and setters

Reply:
Good call out!